contributing/contribution_guide.rst

```diff
@@ -0,0 +1,128 @@
+How to contribute ?
+===================
+
+As Akeneo PIM is an open-source project all contribution is very welcome.
+
+Install and configure Git
+-------------------------
+
+Follow this link from git to install https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+
+Configure
+
+.. code-block:: php
+
+    git config --global user.name "Your Name"
+    git config --global user.email you@example.com
+
+Create an account on github https://github.com/
+
+Create a branch
+---------------
+
+.. note::
+
+    New to git, learn with : https://git-scm.com/book/en/v2
+
+First of all, fork Pim Community development version on https://github.com/akeneo/pim-community-dev (click on the Fork button).
+
+Then clone your fork locally
+
+.. code-block:: php
+
+    $ git clone git@github.com:USERNAME/pim-community-dev.git
+
+Add the upstream repository as a remote
+
+.. code-block:: php
+
+    $ cd pim-community-dev
+    $ git remote add upstream git://github.com/akeneo/pim-community-dev.git
+
+Create a new branch
+
+.. code-block:: php
+
+    $ git checkout -b BRANCH_NAME master
+
+.. note::
+
+    How to name your branch ?
+
+Now, in order to install Akeneo PIM follow this guide http://docs.akeneo.com/latest/developer_guide/installation/installation_workstation.html.
+
+Work on your branch
+-------------------
+
+Before working on a contribution for an Akeneo repository, please read the following code convention
+http://docs.akeneo.com/latest/reference/best_practices/core/conventions.html and coding standard
+http://docs.akeneo.com/latest/reference/best_practices/core/standards.html to be sure to respect all our standard
+and avoid a lot of comments from core developers on your final Pull Request.
+
+For all contributions, please keep in mind that tests are as important as business code.
+Behavior of the application has to be tested with Behat.
+
+.. note::
+
+    How to setup Behat in Akeneo PIM http://docs.akeneo.com/latest/reference/best_practices/core/behat.html
```

Comment:
Best practices and contribution section looks pretty duplicated in fact :) 
Ok to move these pages to contributing section and add a link from the following page to the contributing chapter http://docs.akeneo.com/latest/reference/best_practices/core.html